client/src/app/auth/auth.service.spec.ts

```diff
@@ -94,15 +95,15 @@ describe('Auth services', () => {
       });
     });
 
-    describe('.signInWithGitHub()', () => {
+    describe('.signInWithCredential()', () => {
```

Comment:
Super glad to see those tests.

However, I'd prefer we do not remove the `signInWithGitHub()` API as part of this PR. While we might not need it anymore within the new authentication protocol, we might need it later when we e.g. introduce a UI where people can choose between signing in with GitHub/Google etc.

Reply:
@myspivey fair enough. Thanks for being resistant here!